app/src/test/java/org/simple/clinic/newentry/country/FormFieldFactoryTest.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.newentry.country
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.BuildConfig
+import org.simple.clinic.appconfig.Country
+import org.simple.clinic.newentry.form.AgeField
+import org.simple.clinic.newentry.form.DateOfBirthField
+import org.simple.clinic.newentry.form.DistrictField
+import org.simple.clinic.newentry.form.GenderField
+import org.simple.clinic.newentry.form.LandlineOrMobileField
+import org.simple.clinic.newentry.form.PatientNameField
+import org.simple.clinic.newentry.form.StateField
+import org.simple.clinic.newentry.form.VillageOrColonyField
+import org.threeten.bp.LocalDate
+import org.threeten.bp.format.DateTimeFormatter
+import java.net.URI
+
+class FormFieldFactoryTest {
```

Comment:
The `FormFieldFactory` seems more like a configuration class than a business logic class. This is a class whose output could be defined by a YAML/XML/JSON as well. I don't think this is something we should test.

Reply:
This tests seems fine for now. Let's leave it as it is.

Reply:
Actually, I've removed it.